wp-content/themes/engage/templates/single.twig

```diff
@@ -3,14 +3,18 @@
 {% block content %}
     <div class="post__wrapper wrapper--pad">
         <div class="article__wrapper container container--lg">
-            <article class="article post-type-{{ post.post_type }} {{ post.thumbnail.src ? 'article--has-featured-img'}}" id="post-{{ post.ID }}">
+            {% if post.post_type == "team" or post.post_type == "board" %}
+                <article class="article post-type-{{ post.post_type }}" id="post-{{ post.ID }}">
+            {% else %}
+                <article class="article post-type-{{ post.post_type }} {{ post.thumbnail.src ? 'article--has-featured-img'}}" id="post-{{ post.ID }}">
```

Comment:
Rather than doing an if/else here with the repeated code, just add in the post type checks to the `{{ post.thumbnail.src and post.post_type !== 'team' and post.post_type !== 'board' ? ... }}`. If this is repeated a lot, we may want to update this in one locale with a variable like `showFeaturedImage`.